.github/workflows/lint.yml

```diff
@@ -33,13 +35,17 @@ jobs:
       matrix:
         node-version: [12.x]
```

Comment:
@skjnldsv this is called `node-version` but it´s an array, right? Should we call it `node-versions`? Maybe sometimes it will have more than one 🤔? Not sure.

Reply:
yes it should :)
Though we really don't test against multiple versions as this is really no use for us